docs/release-notes.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Release Notes
+---
+
+## v2.0.0
+
+### New Features
+
+- Support for multiple Azure IoT Edge Gateways with cache and deduplication
+- Support for multiple concentrators with cache and deduplication
+- Support for running LNS on a separate Edge or Cloud device
+- Support for running the LNS on [Windows](https://docs.microsoft.com/en-us/azure/iot-edge/iot-edge-for-linux-on-windows?view=iotedge-2018-06) 
+- Support for [LoRa Basics™ Station](https://github.com/lorabasics/basicstation)
+- Regional support for CN470 and AS923
+- Support for CUPS credential management
+- Support for CUPS concentrator firmware update
+- First class support for Class C devices
+- Observability: integration with Azure Monitor and metrics that are exposed on a Prometheus endpoint
+
+### Breaking Changes
```

Comment:
In this section I would at least add:

- Infrastructure changes: New storage containers for CUPS credentials management and firmware upgrade (more in related [ADRs](link to ADRs)
- Infrastructure changes: @bastbu were there some changes related to deployment of Azure Monitor resources?

Reply:
We need log analytics for the integration with the metrics collector.